docs/en/connector-v2/sink/Mysql.md

```diff
@@ -2,6 +2,10 @@
 
 > JDBC Mysql Sink Connector
 
+## Support Connector Version
```

Comment:
The phrase 'Support Connector Version' is definitely not accurate. Consider using 'Support Mysql Version', 'Support Version', and so on.